common/buildings/hfe_buildings.txt

```diff
@@ -410,4 +526,1210 @@ hfe_building_thoth_module_industry = {
+				has_country_flag = high_mechanicus_retained
+				has_country_flag = hfe_varelse_fedora_governance
+				has_civic = hfe_civic_high_mechanicus
+				has_civic = hfe_civic_overthrown_mechanicus
+			}
+		}
+	}
+
+	allow = {
+		planet = {
+			OR = {
+				has_planet_flag = hfe_varelse_colony
+				has_planet_flag = hfe_colony_beta
+			}
+		}
+		owner = {
+			OR = {
+				has_country_flag = hfe_varelse_primitive_0
+				has_country_flag = high_mechanicus_retained
+				has_country_flag = hfe_varelse_fedora_governance
+			}
+		}
+#		owner = { is_country_type = primitive }
+	}
+
+	country_modifier = {
+		armies_alloys_upkeep_mult = -0.05
+		armies_alloys_cost_mult = -0.05
+	}
+
+	planet_modifier = {
+		armies_alloys_cost_mult = -1
+		armies_alloys_upkeep_mult = -1
+		armies_energy_upkeep_mult = -0.5
+		armies_consumer_goods_cost_mult = -0.9
+		planet_pop_assembly_add = 1
+	}
+	
+	triggered_planet_modifier = {
+		potential = {
+			owner = { is_country_type = primitive }
+			owner = {
+				OR = {
+					has_country_flag = hfe_varelse_primitive_0
+					has_country_flag = high_mechanicus_retained
+					has_country_flag = hfe_varelse_fedora_governance
+#					has_government = hfe_gov_high_mechanicus
+				}
+			}
+			NOT = {
+				owner = { has_ethic = ethic_gestalt_consciousness }
+			}
+		}
+		modifier = {
+			job_hfe_job_godforge_primitive_add = 1
+			job_primitive_laborer_add = 5
+			job_primitive_priest_2_add = 1
+			job_researcher_add = -4 # Negating the Relic stuff
+			job_soldier_add = -1
+		}
+	}
+
+	triggered_planet_modifier = {
+		potential = {
+			owner = { has_country_flag = hfe_varelse_primitive_0 }
+			owner = {
+				OR = {
+					has_civic = hfe_civic_high_mechanicus
+					is_spiritualist = yes
+				}
+				NOT = { has_civic = civic_barbaric_despoilers }
+			}
+		}
+		modifier = {
+			job_noble_add = 1
+		}
+	}
+
+	triggered_planet_modifier = {
+		potential = {
+			owner = { has_country_flag = hfe_varelse_primitive_0 }
+		}
+		modifier = {
+			planet_metallurgists_minerals_upkeep_mult = -0.5
+		}
+	}
+
+	triggered_planet_modifier = {
+		potential = {
+			owner = { is_ai = yes }
+		}
+		planet_districts_upkeep_mult = -0.75
+	}
+
+	triggered_planet_modifier = {
+		potential = {
+			owner = {
+				NOT = {
+					is_country_type = primitive
+					is_primitive = yes
+					has_ethic = ethic_gestalt_consciousness
+				}
+			}
+#			OR = {
+#				owner = { has_country_flag = hfe_varelse_primitive_0 }
+#				owner = { has_country_flag = high_mechanicus_retained }
+#				owner = { has_country_flag = hfe_varelse_fedora_governance }
+#				owner = { has_government = hfe_gov_high_mechanicus }
+#			}
+		}
+		modifier = {
+			job_hfe_job_godforge_overlord_add = 1
+		}
+	}
+
+	triggered_planet_modifier = {
+		potential = {
+			OR = {
+				NOT = { owner = { has_country_flag = hfe_varelse_primitive_0 } }
+				NOT = { owner = { is_country_type = primitive } }
+			}
+			NOT = { owner = { has_ethic = ethic_gestalt_consciousness } }
```

Comment:
you can use nicer scripted trigger here

Reply:
I don't need to do that. This works as it is intended.